cockatrice/src/window_main.cpp

```diff
@@ -319,6 +319,9 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
         case Response::RespClientIdRequired:
             QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\nPlease close and reopen your client to try again."));
             break;
+        case Response::RespContextError:
+            QMessageBox::critical(this, tr("Error"), tr("An internal error has occurred, please try closing and reopening your client and try again. If the error persists try updating your client to the most recent build and if need be contact your software provider."));
```

Comment:
The server returns this response in a lot of places.

On Sun, Aug 9, 2015 at 4:37 PM woogerboy21 <notifications@github.com> wrote:

> In cockatrice/src/window_main.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1350#discussion_r36593326>:
>
> > @@ -319,6 +319,9 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
> >          case Response::RespClientIdRequired:
> >              QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\nPlease close and reopen your client to try again."));
> >              break;
> > +        case Response::RespContextError:
> > +            QMessageBox::critical(this, tr("Error"), tr("An internal error has occurred, please try closing and reopening your client and try again. If the error persists try updating your client to the most recent build and if need be contact your software provider."));
>
> The error message only occurs in the login process and not any were else.
> Could there be a better description, sure. Is it better than before. YES.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1350/files#r36593326>.
>